content/testing-yaml/testing.md

```diff
@@ -7,6 +7,8 @@ aliases:
 ---
 
 Test scripts are added under `scripts` in the [overall architecture](../getting-started/yaml#template), before the build commands.
+You can display test results visually in the build overview if you use expanded form of the script in codemagic.yaml.
+Just include the `test_report` field with a glob pattern matching the test result file location. Supported test report formats are [Junit XML](https://llg.cubic.org/docs/junit/) and Flutter `--machine` report in `.json` format.
```

Comment:
Supported test report formats are [Junit XML](https://llg.cubic.org/docs/junit/) and `.JSON` for Flutter's `--machine` report.

Reply:
fixed in e0d9c8e6222cf49670671fc10c0b345d6644767c